packages/css/README.md

```diff
@@ -49,3 +49,15 @@ To use the compact mode, import the compact css **after** theme css, like so:
 import "@amsterdam/design-system-tokens/dist/index.css";
 import "@amsterdam/design-system-tokens/dist/compact.css";
 ```
+
+## Document styles
+
+Some elements of your document are out of reach for the design system components.
+However, their default styles or common boilerplate approaches can influence on the appearance on the CSS components.
+Apply the following to the document elements:
+
+The font-size must not be changed – e.g. through `html { font-size: 62.5% }` – because the typography system expects a rem to be 16 pixels.
+
+The `ams-body` class must be added to the `<body>` element – it prepares the base layout and sets a background colour.
+
+After the `theme-ams` class has been set, e.g. on the `<body>` as well, bold text will use our extra bold font weight.
```

Comment:
We don’t instruct to add this theme class anywhere. I couldn’t find it on the NLDS website yet, either.

Reply:
Do you mean the `ams-theme` class rendered by our tokens packages? There are two ways to use our tokens in an app:

1. You can import `index.css`, which sets the tokens on `:root`
2. You can import `index.theme.css`, which contains an `ams-theme` class. You can set that class on the body tag for example. This is useful when you switch themes in your app (spacious and compact, or light and dark in the future for example)

We should definitely write docs about that. I don't quite understand why you reuse that `ams-theme` class here though?

Reply:
I agree that it's a good idea to scope the `font-weight` declaration block, but maybe we should use a different class name for it? `ams-theme` is a bit confusing, it's already used for something else.

Reply:
I can’t think of anything better now, and it might be a bit out of scope for this PR as well. I’ve removed it.